lib/components/TokenDropdown.jsx

```diff
@@ -31,15 +33,15 @@ export const COMPOUND_TOKENS = Object.freeze({
       </>
     )
   },
-  // cUsdt: {
-  //   value: 'cUSDT',
-  //   view: (
-  //     <>
-  //       <img src={UsdtSvg} className='inline-block w-6 sm:w-8 mr-3' />
-  //       Tether
-  //     </>
-  //   )
-  // },
+  cUsdt: {
```

Comment:
This was commented out on purpose - apologies, I should have left a note! The tether pools are not working right now and we didn't want anyone wasting gas creating one